### DISCUSSION TITLE
Basic Authentication in ES6 implementation

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#2838] [General]
We are using conductors 3.5.2 and it uses ES6 implementation. How can we pass username and Password in ES6 ?

Conductor.elasticsearch.username is not working in ES6.

**Comment by jxu-nflx** [comment]
Hello @shekarpcs ES6 doesn't support basic auth right now, you could switch to ES7 for support of that: https://github.com/Netflix/conductor/blob/main/es7-persistence/README.md

**Comment by arketec** [comment]
is there a way to make these credentials variables (es7) so to not need to hard-code them in the config.properties?

**Comment by ZergRushJoe** [comment]
when running this in kubenetes we just lock it down with istio policies. we leave es open and limit traffic to only the conductor service and the instance.

**Comment by shekarpcs** [comment]
When will ES7 will be part of main Repo?

We cannot we make ES7 default one. We have many limitations with ES 6 and
may be it is time to move to ES7
